src/Model/Api/Base/StorageZone/AddStorageZone.php

```diff
@@ -50,6 +50,7 @@ public function getBody(): array
                 new AbstractParameter(name: null, type: Type::STRING_TYPE),
             ]),
             new AbstractParameter(name: 'ZoneTier', type: Type::STRING_TYPE),
+            new AbstractParameter(name: 'StorageZoneType', type: Type::STRING_TYPE),
```

Comment:
This option has the values `NotSupported` or `Supported`, but it is not clear what those options mean. Asked bunny.net support for clarification and waiting for further information.